pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -355,7 +353,49 @@ Upgrading catalog applications using Spark Operator can fail when running `dkp u
 
 ### Minio Disk insufficient space when upgrading
 
-When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue, we recommend that you disable the `fluent-bit` Platform Application before upgrading. 
+When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue, we recommend that you disable the `fluent-bit` Platform Application before upgrading.
+
+### Calico not updated during DKP upgrade
+
+When upgrading a DKP cluster, after what seems to be a successful upgrade, the Calico service might not update as expected and, therefor, is still using the old image. The wrong CNI ClusterResourceSet is being generated and not accounting for Flatcar. This issue only impacts Calico, no other add-ons.
```

Comment:
From @cneth 

```suggestion
When upgrading a DKP cluster running on Flatcar OS, you may find that after the upgrade the Calico services were not updated. This occurs because the upgrade procedure is not correctly updating the Flatcar specific CNI ClusterResourceSet(CRS). This issue only impacts the Calico CRS.
```

Reply:
@msdolbey @cmastr - we need to pick one of these two and then make sure they get carried over to the 2.3/2.4/2.5 pages:

https://d2iq.atlassian.net/wiki/spaces/DENT/pages/150831110/DKP+2.5.0+Known+Issues+and+Limitations

Reply:
Gonna commit Matt's then make new changes. We will need a JIRA to track 2.5, or we'll need to copy paste it now before we merge this PR.